src/main/java/org/zalando/nakadi/service/subscription/state/StreamingState.java

```diff
@@ -380,11 +379,13 @@ public void logExtendedCommitInformation() {
                             ", ZkCommitted: " + realCommitted.get(etp) + ")";
                 }).collect(Collectors.joining(", "));
                 getLog().warn("Stale offsets during streaming commit timeout: {}", bustedData);
-                TracingService.logWarning(scope, "Stale offsets during streaming commit timeout: " + bustedData);
+                TracingService.logWarning(getContext().getCurrentSpan(),
+                        "Stale offsets during streaming commit timeout: " + bustedData);
             }
         } catch (NakadiRuntimeException ex) {
             getLog().warn("Failed to get nakadi cursors for logging purposes.");
-            TracingService.logWarning(scope, "Failed to get nakadi cursors for logging purposes.");
+            TracingService.logWarning(getContext().getCurrentSpan(),
```

Comment:
user will definitely not need it, it is quite internal exception, which we could track in the logs